dashboards/TDinsightV3-CN.md

```diff
@@ -0,0 +1,324 @@
+
+### 数据库
+
+![tdinsight-database](../assets/TDinsightV3-5-database.png)
+
+数据库使用情况，对变量 `$database` 的每个值即每个数据库进行重复多行展示。
+
+1. **STables**：超级表数量。
+2. **Total Tables**：所有表数量。
+3. **Tables**：所有普通表数量随时间变化图。
+4. **Tables Number Foreach VGroups**：每个VGroups包含的表数量。
+
+### DNode 资源使用情况
+
+![dnode-usage](../assets/TDinsightV3-6-dnode-usage.png)
+
+数据节点资源使用情况展示，对变量 `$fqdn` 即每个数据节点进行重复多行展示。包括：
+
+1. **Uptime**：从创建 dnode 开始经过的时间。
+2. **Has MNodes?**：当前 dnode 是否为 mnode。
+3. **CPU Cores**：CPU 核数。
+4. **VNodes Number**：当前dnode的VNodes数量。
+5. **VNodes Masters**：处于master角色的vnode数量。
+6. **Current CPU Usage of taosd**：taosd进程的CPU使用率。
+7. **Current Memory Usage of taosd**：taosd进程的内存使用情况。
+8. **Disk Used**：taosd数据目录的总磁盘使用百分比。
+9. **CPU Usage**：进程和系统 CPU 使用率。
+10. **RAM Usage**：RAM 使用指标时间序列视图。
+11. **Disk Used**：多级存储下每个级别使用的磁盘（默认为 level0 级）。
+12. **Disk IO**：磁盘IO速率。
+13. **Net IO**：网络IO，除本机网络之外的总合网络IO速率。
+
+### 登录历史
+
+![登录历史](../assets/TDinsightV3-7-login-history.png)
+
+目前只报告每分钟登录次数。
+
+### TaosAdapter
+
+![taosadapter](../assets/TDinsightV3-8-taosadaper.png)
+
+包含 taosAdapter 请求统计和状态详情。包括：
+
+1. **http_request**: 包含总请求数，请求失败数以及正在处理的请求数
+2. **Http Request Fail**: 处理失败的请求数
+3. **CPU Used**: taosAdapter cpu 使用情况
+4. **Memory Used**: taosAdapter 内存使用情况
+5. **Http Request Inflight**: 每个 endpoint http 请求统计
+6. **Http Status Code**: Http 状态码统计
+
+## 升级
+
+可以通过重新安装 Grafana 插件和重新导入 TDinsight for 3.x 面板的方式升级 TDinsight。
+
+## 卸载
+
+要完全卸载TDinsight v3.x，需要清理以下内容：
+
+1. 删除 Grafana 中的 TDinsight Dashboard v3.x。
+2. 删除 Grafana 中的 Data Source 数据源。
+3. 删除 Grafana 中的 `tdengine-datasource` 插件。
+
+## 整合的Docker示例
+
+```bash
+git clone --depth 1 https://github.com/taosdata/grafanaplugin.git
+cd grafanaplugin
+```
+
+根据需要在 `docker-compose.yml` 文件中修改：
+
+```yaml
+version: "3.7"
+
+services:
+  tdengine:
+    image: tdengine/tdengine:3.0.2.5
+    environment:
+      TAOS_FQDN: localhost
+  grafana:
+    image: grafana/grafana:9.5.20
+    volumes:
+      - ./dist:/var/lib/grafana/plugins/tdengine-datasource
+      - ./grafana/grafana.ini:/etc/grafana/grafana.ini
+      - ./grafana/provisioning/:/etc/grafana/provisioning/
+      - grafana-data:/var/lib/grafana
+    environment:
+      TDENGINE_API: ${TDENGINE_API}
+      TDENGINE_USER: ${TDENGINE_USER}
+      TDENGINE_PASS: ${TDENGINE_PASS}
+      SMS_ACCESS_KEY_ID: ${SMS_ACCESS_KEY_ID}
+      SMS_ACCESS_KEY_SECRET: ${SMS_ACCESS_KEY_SECRET}
+      SMS_SIGN_NAME: ${SMS_SIGN_NAME}
+      SMS_TEMPLATE_CODE: ${SMS_TEMPLATE_CODE}
+      SMS_TEMPLATE_PARAM: "${SMS_TEMPLATE_PARAM}"
+      SMS_PHONE_NUMBERS: $SMS_PHONE_NUMBERS
+      SMS_LISTEN_ADDR: ${SMS_LISTEN_ADDR}
+    ports:
+      - 3000:3000
+volumes:
+  grafana-data:
+```
+
+替换`docker-compose.yml`中的环境变量或保存环境变量到`.env`文件，然后用`docker-compose up`启动Grafana。`docker-compose`
+工具的具体用法参见 [Docker Compose Reference](https://docs.docker.com/compose/)
+
+```bash
+docker-compose up -d
+```
+
+TDinsight 已经通过 Provisioning 部署完毕，请到 http://localhost:3000/d/tdinsight/ 查看仪表盘。
```

Comment:
Remove docker related docs since its just for TDinsight 2.x, and url is not `/d/tdinsight/`